src/server/webpack.config.js

```diff
@@ -29,7 +29,7 @@ const config = {
   module: {
     loaders: [
       {
-        test: /\.js$/,
+        test: /\.jsx?$/,
```

Comment:
Yes. As said by @levithomason, this pattern optionally matches the last letter "x". So it matches both ".js"  and ".jsx"